### ISSUE TITLE
Changing stack order

### ISSUE THREAD

**Issue by dpanster** [kurkle/chartjs-chart-sankey#9]
Hi,

is there a possibility to force a custom stack ordering at a vertical bar group? For example: Set an priority index within a dataset property property?
Like this (with the result, that "Oil" is on top, middle "Coal", bottom "Natural Gas" in the left bar. With highest priority number is on top.):
![Unbenannt](https://user-images.githubusercontent.com/1169555/99631118-54dae200-2a3b-11eb-9b80-5771b5a76eea.png)

------
Currently the element with the highest value is at the top.

Btw: Very cool library. Good work.

**Comment by kurkle** [comment]
This would probably be quite useful feature and should not be too hard to implement.
I think that's all that would need to be altered (in addition to reading the options):

https://github.com/kurkle/chartjs-chart-sankey/blob/4ad829f11359f2469ba6d0c504583bdca2481623/src/layout.js#L148-L161

> Btw: Very cool library. Good work.

Thanks ❤️

**Comment by kurkle** [comment]
No, I'm actually wrong, its not that simple. But would be nice to have though.

**Comment by dpanster** [comment]
Thanks for your quick response and for trying to implement this feature. Have a nice day.

**Comment by musicaldoctor** [comment]
+1 (I think)

We're doing this sankey chart on COVID-19 to track counties and their risk tiers through lockdowns:

![image](https://user-images.githubusercontent.com/1133513/107537353-af0ebd80-6bba-11eb-80e7-8b2f531352b4.png)

And we're not being able to make sure the green bar always stays on top on the fourth iteration, and make sure we control the order of all those stacks on each iteration. Also, can't figure out how to change labels (without changing the JSON source file that feeds the chart).

**Comment by kurkle** [comment]
Initial implementation released in v0.3.0

```js
// the y-order of nodes, smaller = higher
var priority = {
  Oil: 1,
  'Narural Gas': 2,
  Coal: 3,
  'Fossil Fuels': 1,
  Electricity: 2,
  Energy: 1
};
```

![image](https://user-images.githubusercontent.com/27971921/107548484-66193200-6bd7-11eb-91c6-e54154deb755.png)

**Comment by musicaldoctor** [comment]
awesome! is it possible to hide those labels or change them through some sort of callback? if so, how? thanks

![image](https://user-images.githubusercontent.com/1133513/107628327-686fa080-6c58-11eb-8dee-ea4670f7622a.png)

**Comment by kurkle** [comment]
@musicaldoctor added `labels` (similar to `priority`), `borderWidth` and `borderColor`. These are released in v0.3.2.

**Comment by dpanster** [comment]
Well done. Thank You :)
